docs/breadcrumbs/mass-deployment.md

```diff
@@ -0,0 +1,11 @@
+# Mass Deployment
+
+Breadcrumbs are volatile and need slightly more upkeep than Tokens and Canaries themselves. They are relatively short-lived, lasting only until the Canary's settings (or IP) changes. Depending on what was changed, new breadcrumbs may need to be generated and deployed.
+
+To make this easier for you, Breadcrumbs can be mass deployed by making use of read-only Flock API auth keys. This allows you to build them into your automation and deployment scripts.
+
+Read-only Flock API keys can be generated via the "[Flock API Key endpoints](http://localhost:8080/flocks-settings/flock-auth-token.html)" and setting `auth_token_type` to `read-only`.
```

Comment:
Is this the right link? And on line 10.

Reply:
Yes it is, it's to show that they can perform the flock API key generation and deletion actions from the API too. Does it look a bit weird?